README.md

```diff
@@ -16,6 +16,29 @@ force its usage.
 
 ## Building
 
+### Requirements
+
+- CMake 3.5+
+- C++14 compiler
+- vcpkg (C++ package manager)
+- Google Cloud SDK (`gcloud` CLI) - optional, only needed for testing with real GCS buckets
+
+### Checkout
+
+````bash
+# Clone the repository with submodules (REQUIRED)
+git clone --recurse-submodules https://github.com/northpolesec/duckdb-gcs.git
+cd duckdb-gcs
+
+# If you already cloned without submodules, initialize them:
+# git submodule update --init --recursive
+
```

Comment:
Missing a trailing "```" here -- see the formatting at https://github.com/northpolesec/duckdb-gcs/tree/rah/fix-slowness?tab=readme-ov-file#checkout

Also, can you add some notes in the README about how to run the tests & benchmarks? There's stuff about using test credentials in there that is not obvious to me how to set up.

Reply:
Done. I added a note, there's nothing special about auth for those, just `make unittest` and `make benchmark`.